inc/zoo/swar/SWAR.h

```diff
@@ -340,6 +345,11 @@ struct BooleanSWAR: SWAR<NBits, T> {
         return BooleanSWAR(MaskLaneMSB ^ *this);
     }
 
+     // BooleanSWAR as a mask: BooleanSWAR<4, u16>(0x0800).asMask() => SWAR<4,u16>(0x0F00)
+     constexpr auto asMask() const noexcept {
+       return SWAR<NBits,T>(this->m_v - (this->m_v >> (NBits-1)) | this->m_v);
```

Comment:
`s/asMask/MSBtoLaneMask/` ?
We have the luxury of using several lines in implementations since this code began to require C++ 14, and now we're in C++ 17 considering C++ 20.
I believe from these three expressions at least two deserve a name.
Do you not like my style of naming the intermediate expressions?

Reply:
Well, what would you name them?